components/eam/cime_config/usermods_dirs/scm/shell_commands

```diff
@@ -0,0 +1,16 @@
+#!/bin/csh
+# need to use single thread
+foreach component ( ATM LND ICE OCN CPL GLC ROF WAV )
+ ./xmlchange  NTASKS_$component=1,NTHRDS_$component=1
+end
+# Restart capability is not currently supported for SCM
+./xmlchange REST_OPTION=never
+# Modify some parameters for CICE to make it SCM compatible
+./xmlchange CICE_AUTO_DECOMP=FALSE
+./xmlchange CICE_DECOMPTYPE=blkrobin
+./xmlchange CICE_BLCKX=1, CICE_BLCKY=1, CICE_MXBLCKS=1
```

Comment:
Is leaving defaults at `CICE_BLCKX=0,CICE_BLCKY=0,CICE_MXBLCKS=0` okay?
```
This component includes user_mods /gpfs/fs1/home/e3smtest/jenkins/workspace/ACME_Anvil_next/ACME/components/eam/cime_config/usermods_dirs/scm
Adding user mods directory /gpfs/fs1/home/e3smtest/jenkins/workspace/ACME_Anvil_next/ACME/components/eam/cime_config/usermods_dirs/scm
RUN: /lcrc/group/e3sm/e3smtest/scratch/anvil/J/SMS_Ln5.ne4_ne4.F-MMFXX-SCM-ARM97.anvil_gnu.JNextIntegration20210816_024144/shell_commands
FROM: /lcrc/group/e3sm/e3smtest/scratch/anvil/J/SMS_Ln5.ne4_ne4.F-MMFXX-SCM-ARM97.anvil_gnu.JNextIntegration20210816_024144
  errput: usage: xmlchange [-h] [-d] [-v] [-s] [--caseroot CASEROOT] [--append]
                 [--subgroup SUBGROUP] [--id ID] [--val VAL] [--file FILE]
                 [--delimiter DELIMITER] [--dryrun] [--noecho] [-f]
                 [-loglevel LOGLEVEL]
                 [listofsettings]
xmlchange: error: unrecognized arguments: CICE_BLCKY=1, CICE_MXBLCKS=1
```

Reply:
I think that would be ok. We took these values from the SCM scripts repo and it worked on the machine we tested. I'll test using 1 on Cori and Chrysalis.

Reply:
doesn't xmlchange error in Azamat's comment look like xmlchange on Anvil does not take multiple arguments ?

Reply:
good point, I'll just explode the xmlchange commands into single lines for each

Reply:
Looks like xmlchange doesn't like white-space between multiple arguments (same message on Chrysalis too).

Reply:
Ah that explains why the same error was not issue when setting NTASK and NTHRDS. Anyway PR#4468 will fix this.